website/client/components/settings/site.vue

```diff
@@ -241,7 +241,7 @@ export default {
     hello.init({
       facebook: process.env.FACEBOOK_KEY, // eslint-disable-line no-process-env
       google: process.env.GOOGLE_CLIENT_ID, // eslint-disable-line no-process-env
-    });
+    }, {redirect_uri: ''});
```

Comment:
what does this do? If it redirects to the current page it's going to have the same problems as before. I think we should redirect always to the same page which we know is authorized (so either the home page or a specific page used only for this)

Reply:
You don't need redirects for client side oauth. This prevents that and fixes the problem. You can test it locally without this and with.

Reply:
👍